doc/source/ray-core/objects/serialization.rst

```diff
@@ -187,8 +206,8 @@ Below, we demonstrate this behavior on a function with a non-serializable object
 
 The resulting output is:
 
-
-.. code-block:: bash
+.. testoutput::
+    :options: +SKIP
 
     =============================================================
```

Comment:
I got a slightly different output:

```
==========================================================
Checking Serializability of <function test at 0x101193e20>
==========================================================
!!! FAIL serialization: cannot pickle '_thread.lock' object
Detected 1 global variables. Checking serializability...
    Serializing 'lock' <unlocked _thread.lock object at 0x10134d5c0>...
    !!! FAIL serialization: cannot pickle '_thread.lock' object
    WARNING: Did not find non-serializable object in <unlocked _thread.lock object at 0x10134d5c0>. This may be an oversight.
==========================================================
Variable: 

        FailTuple(lock [obj=<unlocked _thread.lock object at 0x10134d5c0>, parent=<function test at 0x101193e20>])

was found to be non-serializable. There may be multiple other undetected variables that were non-serializable. 
Consider either removing the instantiation/imports of these variables or moving the instantiation into the scope of the function/class. 
==========================================================
Check https://docs.ray.io/en/master/ray-core/objects/serialization.html#troubleshooting for more information.
If you have any suggestions on how to improve this error message, please reach out to the Ray developers on github.com/ray-project/ray/issues/
==========================================================
```